src/main/java/org/cactoos/set/Sorted.java

```diff
@@ -54,7 +54,10 @@ public Sorted(final Comparator<T> cmp, final T... array) {
      * @param src An {@link Iterable}
      */
     @SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
-    public Sorted(final Comparator<T> cmp, final Iterable<T> src) {
+    public Sorted(
+        final Comparator<T> cmp,
```

Comment:
@andreoss I believe the comparator should take a `? super T` here too (a comparator is basically a func from T,T to bool :)